services/secret-service/doc/openapi.yaml

```diff
@@ -1,14 +1,16 @@
 openapi: 3.0.0
 info:
   description: "Secrets service is used in the Open Integration Hub for managing secrets used in integration flows."
-  version: "0.0.5"
+  version: "2.2.0"
```

Comment:
Why not 0.0.6?

Reply:
I believe we have agreed in one of the meetings with CES, that our services start with version 1.0.0. As we already have this service running, for us these changes are the next iteration, thus v. 2.x

Reply:
Aren't we talking about the spec version? we don't care about the implementation version here.

Reply:
ok